src/utils/wallet.js

```diff
@@ -25,6 +25,8 @@ const ACCOUNT_ID_REGEX = /^(([a-z\d]+[-_])*[a-z\d]+[.@])*([a-z\d]+[-_])*[a-z\d]+
 export const ACCOUNT_ID_SUFFIX = process.env.REACT_APP_ACCOUNT_ID_SUFFIX || '.test'
 export const ACCOUNT_CHECK_TIMEOUT = 500
 
+const LINKDROP_CONTRACT_ID = 'linkdrop-test-1';
```

Comment:
@Patrick1904 sorry that I missed this earlier, but I think we need to pass this as part of URL, e.g. `fundingContract`.

Reply:
@vgrichina I've added `fundingContract` to URL. E.g.

http://localhost:1234/create/linkdrop-test-1/xGY3TYSTmXEtqAT4C1L5vDdCwHBCmvvUM8XVpRuqHiu3u9ttoDMje3iZTbYFqrrwjGADWkRYNKztxjvctzoazEG